### DISCUSSION TITLE
Do you know how to get the content mathml?

### DISCUSSION THREAD

**Discussion by Jehuty-ML** [roniemartinez/latex2mathml#289] [Ideas]
this package is uesful for getting the presentation mathml! But anyone know how to convert latex or presetational mathml into **content mathml in Python**??  Any help will be Aprreciated!!

**Comment by roniemartinez** [comment]
@Jehuty-ML I decided to wait for other people's inputs on this before making a comment (but unfortunately no one did). 

Content MathML is not supported at the moment and I am estimating that it will need big effort to add this support (plus I am not really familiar with it). I do not have plan to add this right now but probably in the future.

I will still leave this issue open just in case someone decided to help. Thank you.

**Comment by leingang** [comment]
Is this even possible?  LaTeX has some elements that are unambiguous out of context, but many are not.  For instance `|a|` *probably* should be rendered with the `<abs>` element...unless a is a vector and `|a|` means its length, or `a` is a set and `|a|` means its cardinality.  The parser isn't going to know that unless the author of the LaTeX could declare it somehow.
